base_classes/NXdetector.nxdl.xml

```diff
@@ -424,6 +424,24 @@
         <dim index="3" value="k" />
       </dimensions>
     </field>
+  </group>
+
+  <field name="real_time" type="NX_NUMBER" units="NX_TIME">
+    <doc>
+      Real-time of the exposure (use this if exposure time varies for
+      each array element, otherwise use ``count_time`` field).
+	    
+      Most often there is a single real time value that is constant across
+      an entire image frame.  In such cases, only a 1-D array is needed.
+      But there are detectors in which the real time
+      changes per pixel. In that case, more than one dimension is needed.
+    </doc>
+    <dimensions rank="3">
+      <dim index="1" value="i" />
```

Comment:
If same as `polar_angle` then 1:`np`, 2:`i`, 3:`j`

Reply:
@prjemian, @vasole https://github.com/nexusformat/definitions/pull/866/commits/01de4733ed1b2cc488ffc9154de9a4a45c3c5662 okay?

Reply:
@PeterC-DLS Could you add your comment as additional advice in the documentation?

> If same as polar_angle then 1:np, 2:i, 3:j

Reply:
How's this https://github.com/nexusformat/definitions/pull/866/commits/3430099d4974faa61303e0325c01b21f1ffb5836

Reply:
Clear.  Thanks.